docs/templating/static-file-server.html

```diff
@@ -73,6 +73,11 @@ <h1 class="docs-title"><a href="/index.html">KITURA <span class="blue-text">DOCS
           <li class="nested-sidebar-item"><a href="../templating/stencil.html">Stencil</a></li>
           <li class="nested-sidebar-item"><a href="../templating/markdown.html">Markdown</a></li>
         </ul>
+        <li class="sidebar-item collapsible">WebSockets</li>
+        <ul class="nested-sidebar-list content">
+          <li class="nested-sidebar-item"><a href="../websockets/websockets.html">What are WebSockets?</a></li>
+          <li class="nested-sidebar-item"><a href="../websockets/kiturawebsockets.html">Kitura-WebSocket Echo Server</a></li>
```

Comment:
I'm not sure what this menu item needs to be called but we've tried to avoid names with hyphens etc in the menu names in favour of more readable names so I don't like this!